paddle/fluid/operators/crop_op.cc

```diff
@@ -201,13 +201,16 @@ class CropGradOpMaker : public framework::SingleGradOpMaker<T> {
   }
 };
 
+DECLARE_NO_NEED_BUFFER_VARS_INFERENCE(GropNoNeedBufferVarInference, "Y");
```

Comment:
Seems that `X` is also not needed? Please see the implementation of this op. "X" can be removed from the grad op maker.

Reply:
I found https://github.com/PaddlePaddle/Paddle/blob/36afdbd3e11149208ce4d779e4171d4e2c054611/paddle/fluid/operators/crop_op.h#L62

Reply:
My fault. This changing is right.